packages/reaction-accounts/client/templates/inline/inline.html

```diff
@@ -1,6 +1,6 @@
 <template name="loginInline">
   <div class="accounts-dialog accounts-inline">
-    {{#unless currentUser}}
```

Comment:
I didn't remove or change `currentUser`only refactored it and replaced it in this one location where it was incorrect.